CONTRIBUTING

```diff
@@ -35,5 +34,21 @@ documentation.
 * The `ACRN user mailing list`_ is a great place to engage with the
   community, ask questions, discuss issues, and help each other.
 
+
+.. _tsc_members:
+
+Technical Steering Committee (TSC)
+##################################
```

Comment:
Use a line of asterisks for a second-level heading here.

Reply:
Fixed